backend/babel.config.js

```diff
@@ -8,6 +8,10 @@ module.exports = {
       }
     ]
   ],
+  "plugins": [
+    "@babel/plugin-proposal-nullish-coalescing-operator",
```

Comment:
Were these intentionally added? You dont seem to use nullish coalescing or optional chaining in the backend code yet, or do you intend to in the future?

Reply:
Yes I was working with enum in a different branch but added these 2 plugins together